sql/core/benchmarks/JoinBenchmark-jdk21-results.txt

```diff
@@ -2,81 +2,81 @@
 Join Benchmark
 ================================================================================================
 
-OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1018-azure
+OpenJDK 64-Bit Server VM 21.0.4+7-LTS on Linux 6.5.0-1025-azure
 AMD EPYC 7763 64-Core Processor
 Join w long:                              Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Join w long wholestage off                         2147           2166          27          9.8         102.4       1.0X
-Join w long wholestage on                           944            990          40         22.2          45.0       2.3X
+Join w long wholestage off                         2088           2099          15         10.0          99.6       1.0X
+Join w long wholestage on                           918            947          28         22.8          43.8       2.3X
 
-OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1018-azure
+OpenJDK 64-Bit Server VM 21.0.4+7-LTS on Linux 6.5.0-1025-azure
 AMD EPYC 7763 64-Core Processor
 Join w long duplicated:                   Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Join w long duplicated wholestage off              2214           2248          48          9.5         105.6       1.0X
-Join w long duplicated wholestage on                996           1005          10         21.1          47.5       2.2X
+Join w long duplicated wholestage off              1991           1993           3         10.5          94.9       1.0X
+Join w long duplicated wholestage on                911            923          16         23.0          43.4       2.2X
 
-OpenJDK 64-Bit Server VM 21.0.3+9-LTS on Linux 6.5.0-1018-azure
+OpenJDK 64-Bit Server VM 21.0.4+7-LTS on Linux 6.5.0-1025-azure
 AMD EPYC 7763 64-Core Processor
 Join w 2 ints:                            Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Join w 2 ints wholestage off                     148982         149062         112          0.1        7104.0       1.0X
-Join w 2 ints wholestage on                      105434         105515          63          0.2        5027.5       1.4X
+Join w 2 ints wholestage off                     106730         106790          85          0.2        5089.3       1.0X
+Join w 2 ints wholestage on                      105489         105534          40          0.2        5030.1       1.0X
```

Comment:
`Join w 2 ints wholestage on` becomes slower in both Java 17 and 21.

Or, `wholestage off` code becomes faster in this case.